src/EventSubscriber/EventSubscriber.php

```diff
@@ -91,8 +91,8 @@ public function onContentImport(ImportEvent $event) {
    *   Local file path with schema.
    */
   protected function getFile($file_uri) {
-    $local_directory = substr($file_uri, 0, strrpos($file_uri, '/'));
-    $this->fileSystem->prepareDirectory($local_directory, FileSystemInterface::CREATE_DIRECTORY);
+    $local_directory = dirname($file_uri);
+    $this->fileSystem->prepareDirectory($local_directory, FileSystemInterface::CREATE_DIRECTORY | FileSystemInterface::MODIFY_PERMISSIONS);
```

Comment:
I think this is the first time I've ever seen someone use a bitmask inclusive OR.  I had to look it up.